regression/strings/StaticCharMethods01/test.desc

```diff
@@ -1,8 +1,7 @@
-FUTURE
+CORE
 StaticCharMethods01.class
 --refine-strings
 ^EXIT=0$
 ^SIGNAL=0$
 ^VERIFICATION SUCCESSFUL$
 --
-^warning: ignoring
```

Comment:
Some tests were failing because of these warnings, so I've removed them. IIRC I asked @allredj about that and he said that if they cause any problems, they should be removed.